src/main/java/com/example/carservice/repository/CarRepository.java

```diff
@@ -8,4 +8,5 @@
 public interface CarRepository extends JpaRepository<Car, Long> {
 
     List<Car> findAll();
+    Car findCarByModel(String model);
```

Comment:
In the interface, you need to clean up the code and alternate between an empty line and code